### PR TITLE
bitnami/mariadb Add security context setup to the metrics containers

### PR DESCRIPTION
**Pull request by madrisan**
**Description of the change**

This patch add the security context setup to the MariaDB metrics containers.

**Benefits**

This patch allows a deployment to restrict Kubernetes environments where Polaris deny the deployment of containers without a strict containerSecurityContext.

**Possible drawbacks**

There're should be no drawbacks, because for compatibility this feature is disabled by default.

**Additional information**

Successfully tested on Kubernetes with the following configuration:
```
metrics:
  containerSecurityContext:
    enabled: true
    runAsUser: 1001
    runAsGroup: 1001
    readOnlyRootFilesystem: true
    allowPrivilegeEscalation: false
    capabilities:
      drop:
        - ALL
  enabled: true
```

**Checklist** 

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

**Comment by javsalgar** [comment]
Hi! Thank you so much for the PR! Could you bump the minor version of the chart?

**Comment by madrisan** [comment]
Done @javsalgar

File: bitnami/mariadb/Chart.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-mariadb
   - https://github.com/prometheus/mysqld_exporter
   - https://mariadb.org
-version: 10.0.2
+version: 10.0.3
```
Comment (migruiz4) [inline bitnami/mariadb/Chart.yaml:29]: Please bump a minor version (`10.1.0`)

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Hi @madrisan,

Since this PR adds a new value it needs a minor version bump (10.1.0) and the new value to be added in the README.md.

**Review by migruiz4 — APPROVED** [review]
Looks good to me! Thank you for applying the suggested changes.

**Pull request merged**